src/App.css

```diff
@@ -190,3 +190,10 @@ input[type="file"] {
     opacity: 1;
   }
 }
+
+.class-col {
```

Comment:
This should probably have a more specific name; we don't want all the CSS class names to start with "class", and it applies to only a couple of specific "Col"s in the TrackListItem.